agent/bench-scripts/pbench-fio

```diff
@@ -850,6 +869,14 @@ function fio_print_summary() {
 fio_process_options "$@"
 fio_install
 
+# pandas installed only on controller
+if [ "$remote_only" = "n" ] ;then
```

Comment:
Both = and == work the same way, at least when comparing strings in [ ... ] constructs. I use = mostly but as you noticed, I'm not consistent :-) I'll change the == to = throughout.